src/VisualStudio/Core/Def/Implementation/Preview/FileChange.cs

```diff
@@ -80,18 +79,13 @@ private ChangeList ComputeChildren(TextDocument left, TextDocument right, Cancel
 
             diffService = diffService ?? diffSelector.DefaultTextDifferencingService;
 
-            var diff = ComputeDiffSpans(diffService, left, right, cancellationToken);
-            if (diff.Differences.Count == 0)
+            var diff = ComputeDiffSpans(oldText, newText, diffService, cancellationToken);
+            if (!diff.Differences.Any())
```

Comment:
this will allocate enumerator. I don't think Any check whether given IEnumerable support a types that support Count